requirements.txt

```diff
@@ -4,8 +4,8 @@ jmespath
 mock
 molecule
 moleculerize
-openstacksdk
```

Comment:
Do we still need this openstacksdk for future use of openstack service APIs? Or it will be installed in a different place?

Reply:
@phongdly It turns out that "openstacksdk" is a dependency of "shade" so it gets installed anyways.